packages/ui/components/empty-screen/EmptyScreen.tsx

```diff
@@ -22,7 +22,7 @@ export function EmptyScreen({
   Icon?: SVGComponent | IconType;
   avatar?: React.ReactElement;
   headline: string | React.ReactElement;
-  description: string | React.ReactElement;
+  description?: string | React.ReactElement;
```

Comment:
Needed to avoid adding a description in the tea without event types empty screen introduced